staging/src/k8s.io/apiserver/pkg/endpoints/installer.go

```diff
@@ -839,7 +839,10 @@ func (n rootScopeNaming) GenerateLink(req *restful.Request, obj runtime.Object)
 			return "", err
 		}
 	}
-	return n.pathPrefix + url.QueryEscape(name) + n.pathSuffix, nil
+	if len(n.pathSuffix) == 0 {
+		return n.pathPrefix + "/" + url.QueryEscape(name), nil
+	}
+	return n.pathPrefix + "/" + url.QueryEscape(name) + "/" + n.pathSuffix, nil
```

Comment:
I tested this pr and I think @deads2k is right.
See code here: 
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/endpoints/installer.go#L389
We already have one "/" before suffix.
This is my test result:
$curl -s http://172.16.116.128:8080/api/v1/nodes/kubenet-02/status | grep selfLink
    "selfLink": "/api/v1/nodes/{name}/status/kubenet-02//status",